tests/basic_integration/fns.sh

```
@@ -0,0 +1,138 @@
+    yarn enclave aggregator start \
+      --tag "$name" \
+      --config "$SCRIPT_DIR/lib/$name/config.yaml" \
+      --pubkey-write-path "$SCRIPT_DIR/output/pubkey.bin" \
+      --plaintext-write-path "$SCRIPT_DIR/output/plaintext.txt" & echo $! > "/tmp/enclave.${ID}_${name}.pid"
+
+    ps aux | grep aggregator
+}
+
+kill_proc() {
+  local name=$1
+  local pid=$(ps aux | grep 'enclave' | grep "\--tag $name" | awk '{ print $2 }')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)